rom/dev/README.md

```diff
@@ -16,9 +16,7 @@
 
 
 ## 2. Spec Opens
```

Comment:
recommend just deleting the section since it will be empty.

Reply:
Updated.